### PR TITLE
replaced HTTPClient.HTTPC with Requests.jl

### PR DESCRIPTION
**Pull request by keith-mason-conning**
Since AWS.jl has been updated to use Requests.jl, continuing to use HTTPC in DynamoDB.jl is producing conflicts.  This pull request will resolve those conflict warnings.

### CODE REVIEW

**Comment by keith-mason-conning** [comment]
BTW, my group is starting to use this package somewhat extensively.  As such, we have identified a number of bugs we are planning to fix and submit pull requests for.  @dls, can you please let me know if you are able to help us get the fixes merged and tagged?

Thanks.

**Comment by dls** [comment]
Hi s3maki,

Thanks for submitting the issue, I don't check github very often so thanks for the ping. Checking what happened with CI then merging.

Thanks,

File: src/dynamo_requests.jl [hunk 1]
```diff
@@ -63,11 +63,11 @@ function signature_version_4(env, service, method, host, action, payload)
...
-    headers = [("Authorization", authorization_header),
-               ("X-Amz-Date", amzdate),
-               ("Host", host),
-               ("Content-Type", "application/x-amz-json-1.0"),
-               ("X-Amz-Target", "DynamoDB_20120810.$action")]
+    headers = Dict("Authorization" => authorization_header,
+               "X-Amz-Date" => amzdate,
+               "Host" => host,
+               "Content-Type" => "application/x-amz-json-1.0",
+               "X-Amz-Target" => "DynamoDB_20120810.$action")
```
Comment (dls) [inline src/dynamo_requests.jl:70]: (I know this somewhat off topic for this request, but still:) a dict here is soo much better

**Comment by dls** [comment]
https://github.com/dls/DynamoDB.jl/commit/be1da03c37232eacf995c78dfb34f40947c1a187 was the issue blocking integration tests

**Pull request merged**